template.yaml

```diff
@@ -96,15 +96,15 @@ Resources:
               title: Lookup DOI URL
               type: object
               properties:
-                doiUrl:
+                doi:
                   type: string
                   description: DOI URL
               required:
                 - doiUrl
   NvaDoiFunction:
     Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
     Properties:
-      CodeUri: .
+      CodeUri: ./build/libs/nva-doi-lambda-1.0-SNAPSHOT.jar
```

Comment:
Why are we doing this?

Reply:
Sorry. I forgot to change it back. But, how do you test locally?

Reply:
reversed change to the  original value